### ISSUE TITLE
Unable to Replicate Results in Table 2 for XRF55 dataset

### ISSUE THREAD

**Issue by kanji1011** [xyanchen/X-Fi#2]
I am unable to replicate results for XRF55 dataset when I train from scratch, am getting only 70.55. However, the pre-trained checkpoint gives the results in table 2. What am I doing wrong, are there different hyperparameter settings I should use? Any help is appreciated

**Comment by xyanchen** [comment]
Hi,

As discussed in Table 3 of the paper, the modality existence probabilities in the modality existence list play a crucial role in achieving desirable model performance. You may consider adjusting these probabilities during model tuning.

In addition, incorporating a learning rate scheduler into the training function, absent in the released version of X-Fi, could further enhance training stability and performance.

Hope the above suggestions could solve your problem.

BR
